### PR TITLE
fix: syndicate jobs now have the type

### PR DESCRIPTION
**Pull request by Mettwasser**
### What did you fix? 
Syndicate jobs now have the "type" attribute, which was missing

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[Yes]**
- Have I run the linter? **[Yes]**
- Is is a bug fix, feature request, or enhancement? **[Bug Fix]**

### CODE REVIEW

File: src/worldstate/models/arbitration.rs [hunk 1]
```diff
@@ -45,7 +45,17 @@ mod test {
...
         match client.fetch::<Arbitration>().await {
             Ok(_arbitration) => Ok(()),
-            Err(why) => Err(why),
+            Err(why) => {
+                if let ApiError::ApiError(error) = why {
+                    if error.code == 404 {
+                        Ok(())
+                    } else {
+                        Err(ApiError::ApiError(error))
+                    }
```
Comment (TobiTenno) [inline src/worldstate/models/arbitration.rs:54]: shouldn't need this anymore cause of wfcd/warframe-worldstate-parser#531

File: src/worldstate/models/faction.rs [hunk 2]
```diff
@@ -11,5 +11,7 @@ enum_builder! {
     Tenno,
     Narmer,
     Crossfire,
-    Murmur = "The Murmur"
+    Murmur = "The Murmur",
+    :"Honestly, no idea what this is"
+    FcMitw = "FC_MITW"
```
Comment (TobiTenno) [inline src/worldstate/models/faction.rs:16]: bruh, MITW? Man in the Wall

File: src/worldstate/models/arbitration.rs (hunk 1 above)
Comment (Mettwasser) [reply src/worldstate/models/arbitration.rs:54]: It still error'd with a 404 :/

Comment (TobiTenno) [reply src/worldstate/models/arbitration.rs:54]: hmm, interesting. ok.

File: src/worldstate/models/syndicate_mission.rs [hunk 3]
```diff
@@ -14,6 +14,8 @@ model_builder! {
     :"The Reward Pool of the Bounty"
     pub reward_pool: Vec<String>,
 
+    :"The name of the syndicate job"
+    pub job_name: String = "type",
```
Comment (TobiTenno) [inline src/worldstate/models/syndicate_mission.rs:18]: stop renaming stuff, dude this makes it really hard for me to know what you're looking for when you tell me there's an issue with a rust field....

Comment (Mettwasser) [reply src/worldstate/models/syndicate_mission.rs:18]: type is a reserved keyword, in addition 'type' must be the single worst name you can give to a field like this

Comment (TobiTenno) [reply src/worldstate/models/syndicate_mission.rs:18]: it's literally called `jobType` in DE's source data. i'm not gonna keep the class prefix when i translate it, bud. 

https://github.com/WFCD/warframe-worldstate-parser/blob/master/lib/models/SyndicateJob.js#L116C32-L116C44

Comment (TobiTenno) [reply src/worldstate/models/syndicate_mission.rs:18]:
```suggestion
    pub job_type: String = "type",
```

since my name is dumb, use DE's dumb name

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**